CMakeLists.txt

```diff
@@ -179,8 +179,8 @@ if(JSONCPP_WITH_PKGCONFIG_SUPPORT)
         DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig")
 endif()
 
+include(CMakePackageConfigHelpers)
```

Comment:
Why is this change necessary?

Reply:
yeah, that is not needed actually, removing it